MoreLinq.Test/MoreLinq.Test.csproj

```diff
@@ -78,7 +83,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
```

Comment:
I believe it's because I've added the T4 template to the test project. I don't believe it's 'needed' as far as building is concerned, but I *think* it's basically a tag used by the IDE to expose certain things for the project.

Reply:
I thought that's what the service in the following line was but seems [`82a7f48d-3b50-4b1e-b82e-3ada8210c358` is for tests](https://stackoverflow.com/a/22917902/6682) so this one must be for T4. I was just surprised it wasn't already there since we've been using T4 since a long time.

Reply:
> since we've been using T4 since a long time.

Not in MoreLinq.Test project, until now

Reply:
Ah, that certainly explains it, @Orace!

Reply:
This means we'll need [`tt.sh`](https://github.com/morelinq/MoreLINQ/blob/b77df70598ab84c28cd43dcf74594024b6d575e1/MoreLinq/tt.sh)/[`tt.cmd`](https://github.com/morelinq/MoreLINQ/blob/b77df70598ab84c28cd43dcf74594024b6d575e1/MoreLinq/cmd.sh) to cover the test project as well now.